src/olympia/devhub/utils.py

```diff
@@ -195,6 +196,25 @@ def __init__(self, file_, addon=None, listed=None, final_task=None):
             try:
                 addon_data = parse_addon(file_, minimal=True)
                 is_mozilla_signed = addon_data.get('is_mozilla_signed_extension', False)
+                if theme_specific and addon_data['type'] != amo.ADDON_STATICTHEME:
```

Comment:
Where is the opposite case handled (`not theme_specific and addon_type['type'] == amo.ADDON_STATICTHEME`)?  Can we have a comment if there's a reason why it's not in the same function?

Reply:
The opposite case is not handled, because we don't care - if someone submits a theme through the non-theme workflow, that's fine - they'll just get the 2FA restrictions when it's added later, but it should work.

Because the non-theme workflow uses the current URLs (that have been used for a while for all add-ons) it's probably best not to restrict that flow at all.

Reply:
94a0173fda47feca3c3c56cfddafa47bf4816f08